opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingDynamicFeature.java

```diff
@@ -197,6 +201,18 @@ public Builder withSkipPattern(String skipPattern) {
             return this;
         }
 
+        /**
+         * @param joinExistingActiveSpan If true, any active span on the current scope will be used as the parent span.
+         *                               If false, and a parent span can be extracted from the HTTP request, that span
+         *                               will be used as the parent span instead. <br />
+         *                               Default is false.
+         * @return builder
+         */
+        public Builder withJoinExistingActiveSpan(boolean joinExistingActiveSpan) {
```

Comment:
Are you happy with the naming of this? I couldn't think of much better, but perhaps it could be improved.